bamboo.sh

```diff
@@ -44,7 +44,7 @@ GetGHRepo pbcore _deps
 GetGHRepo PacBioTestData _deps
 
 echo "## Fetch submodules"
-git submodule update --init --remote --depth 1
+git submodule update --init --remote
```

Comment:
What happened? `--depth 1` now breaks **pb-git** too? Did you rebase a submodule?